scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/nn/SoftMax.scala

```diff
@@ -32,46 +33,17 @@ import scala.reflect.ClassTag
  * where shift = max_i(x_i).
  */
 @SerialVersionUID(- 7842335603491194236L)
-class SoftMax[T: ClassTag](var pos: Int = 1)(implicit ev: TensorNumeric[T])
+class SoftMax[T: ClassTag]()(implicit ev: TensorNumeric[T])
```

Comment:
you should probably replace this class with `dllib.keras.layers.internal.InternalSoftMax`

Reply:
Ok. So we will move the InternalSoftMax implementation to nn.softmax and delete `InternalSoftMax` class?

Reply:
yes